app/src/main/java/com/poupa/vinylmusicplayer/service/MusicService.java

```diff
@@ -798,10 +791,8 @@ public void removeSong(int position) { // better to test is playing here and hav
 
         int newPosition = playingQueue.remove(position);
         if (newPosition != -1) {
-            if (isPlaying)
-                playSongAt(newPosition);
-            else
-                setPosition(newPosition);
+            if (isPlaying) {playSongAt(newPosition, false);}
+            else {setPosition(newPosition);}
```

Comment:
Why adding one liner here ?

Reply:
Personal habit. I do prefer having explicit {} all the time.